Resources/Prototypes/_NF/Shipyard/greenthumb.yml

```diff
@@ -0,0 +1,29 @@
+- type: vessel
+  id: Greenthumb
+  name: NC Greenthumb
+  description: A self-sufficient, eco-friendly bar and restaurant, featuring its own hydroponics garden.
+  price: 46000
+  category: Medium
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/greenthumb.yml
+
+- type: gameMap
+  id: Greenthumb
+  mapName: 'NC Greenthumb'
```

Comment:
Remove "NC" from the napName as NC should not appear in the IFF Tag.

Reply:
Sorry about that. Fixed in [b3e6d092](https://github.com/new-frontiers-14/frontier-station-14/pull/796/commits/b3e6d0928ed5a306debd178d5b3d5aeaa1f2e592)